modules/tensor_mechanics/src/userobjects/CrystalPlasticityStateVarRateComponentGSS.C

```diff
@@ -59,16 +64,42 @@ CrystalPlasticityStateVarRateComponentGSS::calcStateVariableEvolutionRateCompone
   Real qab;
   Real a = _hprops[3]; // Kalidindi
 
+  // the intial critical resolved shear stress:
+  Real tau_0 = 0.0;
+  // Kalidindi -> tau_0 = 0
+  // classic voce -> tau_0 = tau_y at time 0
+  if (_hprops.size() > 4)
+  {
+    tau_0 = _hprops[4];
+  }
+
+  // compute denomintor of the hardening rule
+  Real delta_tau = tau_sat - tau_0;
+
+  // get the number of coplanar slip systems for BCC or FCC
+  unsigned int nSSperPlane = 0;
```

Comment:
I'd like to see this variable name written out fully so that the meaning is clear everywhere in the code. Additionally, could you instead set this variable in the constructor rather than declaring it every time?

Reply:
I agree, however this will produce the following  name: numberOfSlipSystemPerPlane which seems to be a little to long. However I can do this if you think this is better.

Reply:
There's no such thing as too long in the MOOSE world!

Reply:
Please do, or perhaps `numberSlipSystemsPerPlane` (or `_numberSlipSystemsPerPlane` if you use a member variable in the constructor) so that users won't have to guess at the meaning. Thanks!

Reply:
Per http://mooseframework.org/moose/documentation/framework_development/code_standards.html camel case is reserved for method names. So it should be `number_slip_systems_per_plane` or `n_slip_systems_per_plane` (we use just `n` to prefix a size  variable).

Reply:
whoops, thanks for the catch on correct case @dschwen